client/components/pickup/OrgCard.tsx

```diff
@@ -28,14 +26,11 @@ const OrgCard = (props: Props) => {
 		contactDetail,
 		lastUpdated
 	} = props.orgDetails;
+	const acceptedItems = props.acceptedItems;
```

Comment:
can use destructuring of props
```suggestion
	const { acceptedItems, notAcceptedItems  } = props;
```